args.go

```diff
@@ -114,9 +120,11 @@ func (a *Args) AppendBytes(dst []byte) []byte {
 	for i, n := 0, len(a.args); i < n; i++ {
 		kv := &a.args[i]
 		dst = AppendQuotedArg(dst, kv.key)
-		if len(kv.value) > 0 {
+		if ArgsHasValue == kv.noValue {
```

Comment:
Sorry one last change, can you swap this one around as well.

Reply:
done :)